index.js

```diff
@@ -36,17 +36,13 @@ const { spawn } = require('child_process');
     '-f', 'image2pipe',
     '-r', '30',
```

Comment:
You are specifying 30 fps as a target frame rate both here and on line 73 - but on line 73, it is very implicitly encoded as the result of 1000/30.

Could you define a constant and reuse it everywhere? I am sorry if this causes weird floating point issues.
This does look very easy to overlook if the next programmer just does a "CTRL-F" for "30".

Reply:
I changed that part of the code already and will ask for a review in another PR.